build.js

```diff
@@ -102,6 +102,14 @@ function copyFiles() {
       files.push([`${dawnOutputDir}/libshaderc_spvc.dylib`, targetDir]);
       files.push([`${dawnOutputDir}/libshaderc.dylib`, targetDir]);
     }
+    else if (platform === "linux") {
+      files.push([`${dawnOutputDir}/libdawn_native.so`, targetDir]);
+      files.push([`${dawnOutputDir}/libc++.so`, targetDir]);
+      files.push([`${dawnOutputDir}/libdawn_proc.so`, targetDir]);
+      files.push([`${dawnOutputDir}/libdawn_wire.so`, targetDir]);
+      files.push([`${dawnOutputDir}/libshaderc_spvc.so`, targetDir]);
+      files.push([`${dawnOutputDir}/libshaderc.so`, targetDir]);
```

Comment:
This is for a later time, but by taking advantage of the gn arg `dawn_complete_static_libs=true` it is probably possible to reduce this to just two dlls/dylibs/sos (libdawn_native + libdawn_proc). We also might add a target to create a single standalone dll that combines them.

Reply:
Nice. I just used the existing pattern.

Reply:
Thanks @kainino0x for reviewing :) My laptop died last week, so I'm a bit delayed on continuing with this project

Reply:
no problem! I didn't do a full review, only skimmed, but always feel free to @ me if you have questions.

Reply:
I think Felix mentioned that the libshaderc dependency might be able to be dropped?

Reply:
That sounds possible to me. You should try it out. (You will definitely need `libshaderc_spvc` though.)

I misspoke earlier: `dawn_complete_static_libs` does not yet allow you to use only `libdawn_native` + `libdawn_proc`. I have not landed [the patch](https://dawn-review.googlesource.com/c/dawn/+/15481) yet.

Reply:
I notice the patch dropped libshaderc.

Reply:
If you're talking about my patch, that's because it rolled all of the dependencies (including libshaderc_spvc) together into the libdawn_native binary.

Reply:
Yes, but libshaderc (not included) is independent of libshaderc_spvc (included).
Just need to confirm that libshaderc is not a dependency for WebGPU (only uses libshaderc_spvc, which makes sense).